src/app/components/Columns/BookmarkAddColumn.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import styled from 'styled-components'
+
+// Visualisation of add column block
+const BookmarkAddColumn = ({ className, value, onChange, onClick, onKeyPress }) => (
+  <div className={className}>
+    <input name='addColumn'
+      type='text'
+      placeholder='Enter Name'
+      value={value}
+      onChange={onChange}
+      onKeyPress={onKeyPress} />
+    <button onClick={onClick}>
+      <svg viewBox='0 0 35 35' version='1.1' fill=''><g id='' fill=''><path d='M 16 3 C 8.832031 3 3 8.832031 3 16 C 3 23.167969 8.832031 29 16 29 C 23.167969 29 29 23.167969 29 16 C 29 8.832031 23.167969 3 16 3 Z M 16 5 C 22.085938 5 27 9.914063 27 16 C 27 22.085938 22.085938 27 16 27 C 9.914063 27 5 22.085938 5 16 C 5 9.914063 9.914063 5 16 5 Z M 15 10 L 15 15 L 10 15 L 10 17 L 15 17 L 15 22 L 17 22 L 17 17 L 22 17 L 22 15 L 17 15 L 17 10 Z ' fill='' /></g></svg>
```

Comment:
When you look at the webpack config, you see an entry for loading svg files for raw content. so you are able to do:

`<div dangerouslySetInnerHTML={{ __html: require("../path/to/assets/icon.svg")}} />`

Then you have the possibility to reuse your icons without copying the content of every svg